databases/ClickHouse-server/Portfile

```diff
@@ -0,0 +1,84 @@
+long_description    ${description}
+
+categories          databases
+maintainers         @slarew openmaintainer
+license             public-domain
+platforms           darwin
+homepage            https://github.com/ClickHouse/ClickHouse
+master_sites
+supported_archs     noarch
+
+depends_run         port:ClickHouse
+
+distfiles
+
+use_configure       no
+build               {}
+
+add_users clickhouse group=clickhouse shell=/bin/sh realname=ClickHouse
+
+installs_libs       no
+
+#destroot.violate_mtree yes
+
+destroot {
+    xinstall -d -m 0755 -o root -g wheel ${destroot}/Library/LaunchDaemons
+    xinstall -m 0755 -o root -g wheel ${filespath}/org.macports.clickhouse-server.plist ${destroot}/Library/LaunchDaemons
+    reinplace "s|@@PREFIX@@|${prefix}|g" ${destroot}/Library/LaunchDaemons/org.macports.clickhouse-server.plist
+}
+
+post-activate {
+    xinstall -d -b -m 0750 -o clickhouse -g clickhouse ${prefix}/var/db/clickhouse
+    xinstall -d -b -m 0750 -o clickhouse ${prefix}/var/log/clickhouse-server
+    xinstall -d -b -m 0750 -o clickhouse ${prefix}/var/log/clickhouse-keeper
+
+    if {![file exists ${prefix}/etc/clickhouse-server/config.xml]} {
+        file copy ${prefix}/etc/clickhouse-server/config.xml.dist ${prefix}/etc/clickhouse-server/config.xml
+    }
+    if {![file exists ${prefix}/etc/clickhouse-server/users.xml]} {
+        file copy ${prefix}/etc/clickhouse-server/users.xml.dist ${prefix}/etc/clickhouse-server/users.xml
+    }
+    if {![file exists ${prefix}/etc/clickhouse-keeper/keeper_config.xml]} {
+        file copy ${prefix}/etc/clickhouse-keeper/keeper_config.xml.dist ${prefix}/etc/clickhouse-keeper/keeper_config.xml
+    }
+    if {![file exists ${prefix}/etc/clickhouse-client/config.xml]} {
+        file copy ${prefix}/etc/clickhouse-client/config.xml.dist ${prefix}/etc/clickhouse-client/config.xml
+    }
+}
+
+pre-deactivate {
+    catch { system "launchctl bootout system/org.macports.clickhouse-server" }
+    catch { system "launchctl disable system/org.macports.clickhouse-server" }
+}
+
+post-deactivate {
+ui_msg {
+The ClickHouse service's data and configuration were not deleted.
+  database: ${prefix}/var/db/clickhouse
+  logs: ${prefix}/var/log/clickhouse-*
+  configuration: ${prefix}/etc/clickhouse-*
+You may delete those files if they are no longer desired.
+
+The ClickHouse service user and group were not deleted. If you delete the
+database and logs, consider also deleting the service user and group.
+  $ sudo dscl . -delete /Users/clickhouse
+  $ sudo dscl . -delete /Groups/clickhouse
+}
+}
+
+notes {
+    A system service for running clickhouse server has been installed. It is disabled by default. Execute the following commands to start it, and to cause it to launch at startup:
+
+        sudo launchctl enable system/org.macports.clickhouse-server
+        sudo launchctl bootstrap system /Library/LaunchDaemons/org.macports.clickhouse-server.plist
```

Comment:
I haven't seen `launchctl`'s `enable` or `bootstrap` commands before. How is this different from the way that MacPorts loads the plists it generates (with `load`)?